JASP-Engine/JASP/R/principalcomponentanalysis.R

```diff
@@ -18,7 +18,8 @@
 
 .DotIfNULL <- function(x){
 
```

Comment:
@hummyhummy .DotIfNull to a common pcaEfa

Reply:
changed to .DotIfNullPCAEFA